content/webapp/pages/index.tsx

```diff
@@ -106,20 +109,23 @@ export const getServerSideProps: GetServerSideProps<
     order: 'asc',
   });
 
-  const [exhibitionsQuery, eventsQuery, articlesQuery, pageDocument] =
+  const [articlesResponse, exhibitionsQuery, eventsQuery, pageDocument] =
     await Promise.all([
+      articlesResponsePromise,
       exhibitionsQueryPromise,
       eventsQueryPromise,
-      articlesQueryPromise,
       pagePromise,
     ]);
 
   // The homepage should always exist in Prismic.
   const page = transformPage(pageDocument as RawPagesDocument);
 
-  const articles = transformQuery(articlesQuery, transformArticle);
-  const jsonLd = articles.results.map(articleLd);
-  const basicArticles = articles.results.map(transformArticleToArticleBasic);
+  const articles = transformPaginatedResults(
+    articlesResponse,
+    transformContentApiArticle
+  ).results;
```

Comment:
What's the main difference with the current API response and can we make the change in the API instead in a way that would suit wherever it is used?

Reply:
I'm not sure I agree about continuing to use the older/existing components. As we're looking at using the Content API more and more in the future and have built its responses the be in the exact shape we need, being able to do away with old components is a nice step towards a more aligned codebase. Plus, having the same card component used for stories across would simplify debugging and implementing styling changes.

Does using `StoriesGrid` instead of `CardsGrid/StoryPromo` cause styling issues? Again, as this was mostly built with Search in mind, there might be growing pains, happy to chat on a call?

Reply:
I don't think `StoriesGrid` can handle the cards at the top of the stories page (I think if we don't reuse the `StoryPromo` we'd need a like-for-like replacement component?

Reply:
As discussed, this will be split into a different ticket to be referenced here 🙏 
This ticket which will remove the need for `content/webapp/services/wellcome/transformers/paginated-results.ts` and `content/webapp/services/wellcome/transformers/articles.ts`, I therefore won't do much code review on those!